### ISSUE TITLE
Clipboard interoperability with RPG Maker Editor (Event script only)

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Editor-Qt#120]
For everything makes no sense imo (lots of work) but for Script copy-paste there could be use-cases.

Need to figure out the format first, shouldn't be too difficult with a clipboard viewer.

Bonus points when copy-paste from/to editor in Wine works.

**Comment by CherryDT** [comment]
Example:
![image](https://user-images.githubusercontent.com/6010463/50104395-31450680-022a-11e9-9978-9349d4f5675c.png)

**Comment by fmatthew5876** [comment]
Clipboard chunk description probably belongs in liblcf.

**Comment by CherryDT** [comment]
Ah for event _commands_, sorry, overlooked that.
The format is even simpler there.
![image](https://user-images.githubusercontent.com/6010463/50104959-a8c76580-022b-11e9-96fe-932199aee058.png)


However I realize now that the format is not exactly the same as in the files - the event commands have chunk IDs in the LMU/LDB, don't they? I'm a bit confused now because I was pretty sure it uses the same function to read from/write to stream, but in the clipboard the event commands are without chunk IDs (other things like the event data above is with IDs)

For the event commands, it's just the following data for each event line, one after another:

* Command ID (BER)
* Tree depth (BER)
* String parameter (BER length + String data)
* Amount of int parameters (BER)
* Int parameters (BERs)

**Comment by Ghabry** [comment]
Wut? Same as LCF-chunks? Oh, in that case we could support more xD.

**Comment by Ghabry** [comment]
Oh and thanks for this InsideClipboard tool, I already searched for a clipboard reader and couldn't find one ^^

**Comment by MarianoGnu** [comment]
i didn't even know you could copypaste event commands on RM xD

**Comment by fmatthew5876** [comment]
Yes I checked in RE tools and RPG Maker editor just sends and receives the binary event stream.

RPG editor has 3 separate sets of functions: battle, common, and map. These are probably just member functions of the 3 widgets, but haven't dug too deeply.

For receiving, RPG editor receives the whole event stream, then checks and removes any event command id codes which are incompatible. So this is what prevents you from copy pasting battle event code into map events etc..
